dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+PROJECT_NAME=zammad-dev
```

Comment:
Unused? Maybe you wanted to use this as stack name? Then perhaps move it to .env?